uint16 x_move # 1=0.0381mm
uint16 y_move # 1=0.0381mm
uint16 yaw    # 360度/65535*yaw
uint8 grasp   # 90度 /255  *grasp
uint8 lift    # 0/1